atomate/qchem/firetasks/parse_outputs.py

```diff
@@ -77,9 +77,6 @@ def run_task(self, fw_spec):
             output_file=output_file,
             multirun=multirun)
 
-        if "tags" in fw_spec:
-            task_doc.update({"tags": fw_spec["tags"]})
-
```

Comment:
Hey @samblau , in this PR I'm modifying the `add_tags` powerup to be more general and work properly with `QChemToDb` (see #576 . That means the lines removed here will be redundant, and if left in place could result in duplicate tags in the task doc.

I wanted to flag you because IF members of your team are in the habit of directly setting tags in the Firework (i.e. `<Firework>["_spec"]["tags'] = "some_tag"` and then expecting tags to go into the task doc, that functionality will break with this change. Instead, they'd need to use `atomate.common.powerups.add_tags`. Are you OK with that?

As an alternative, I could add some logic that checks to see if 'tags' is listed in 'additional_fields', and if not, passes the `fw_spec` tags through, preserving the current functionality. Please let me know your preference.

Reply:
This is fine with me. FYI @espottesmith

Reply:
Not an issue for any of my workflows either.